### PR TITLE
[Feature] iOS extension test app

### PR DESCRIPTION
**Pull request by Jamminroot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [X] Are tests passing locally?
* [X] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

Extension UI

[AB#66880](https://dev.azure.com/msmobilecenter/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/66880)

### CODE REVIEW

File: Sasquatch/SasquatchSwift Today Extension/TodayViewController.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+class TodayViewController: UIViewController, NCWidgetProviding {
+  
+  @IBOutlet weak var extensionLabel: UILabel!
+  
+  var didStartAppCenter = false;
+  override func viewDidLoad() {
+    
+    super.viewDidLoad()
+    extensionLabel.text = "Run #\(UUID().uuidString)"
+    if (!didStartAppCenter){
```
Comment (guperrot) [inline Sasquatch/SasquatchSwift Today Extension/TodayViewController.swift]: format needed

File: Sasquatch/SasquatchSwift-Preview-Extension/Info.plist [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jaelim-ms) [inline Sasquatch/SasquatchSwift-Preview-Extension/Info.plist]: Reuse `Sasquatch/SasquatchSwift-Extension/Info.plist`. You can set Info.plist path in Build Settings.

File: Sasquatch/SasquatchSwift-Extension/ExtenstionViewController.swift [hunk 3]
```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import UIKit
```
Comment (jaelim-ms) [inline Sasquatch/SasquatchSwift-Extension/ExtenstionViewController.swift]: How do we sort import in Swift?

**Comment by MatkovIvan** [comment]
And one more suggestion about UX:
Screen Shot 2019-09-09 at 15 26 22

**Comment by jaelim-ms** [comment]
I'm not sure it is caused by this extension implementation but when I build SasquatchSwift I have a build error.
```
error: Cycle in dependencies between targets 'AppCenterAnalytics iOS Framework' and 'AppCenterData iOS Framework'; building could produce unreliable results.
Cycle path: AppCenterAnalytics iOS Framework → AppCenterAuth iOS Framework → MSAL (iOS Static Library) → IdentityCore iOS → SasquatchSwift Extension → AppCenterData iOS Framework → AppCenterAnalytics iOS Framework
Cycle details:
Target build order preserved because “Parallelize Build” is off

→ Target 'AppCenterAnalytics iOS Framework': Libtool /Users/jaelim/Library/Developer/Xcode/DerivedData/AppCenter-ggabztdbmpzxaxfjxisclsawjhbn/Build/Products/Debug-iphonesimulator/AppCenterAnalytics.framework/AppCenterAnalytics normal x86_64
→ Target 'AppCenterAuth iOS Framework'
→ Target 'MSAL (iOS Static Library)'
→ Target 'IdentityCore iOS' has target dependency on Target 'SasquatchSwift Extension' due to target order in a “Target Dependencies” build phase or the scheme
→ Target 'SasquatchSwift Extension': Ditto /Users/jaelim/Library/Developer/Xcode/DerivedData/AppCenter-ggabztdbmpzxaxfjxisclsawjhbn/Build/Intermediates.noindex/Sasquatch.build/Debug-iphonesimulator/SasquatchSwift Extension.build/DerivedSources/Sasquatch-Swift.h /Users/jaelim/Library/Developer/Xcode/DerivedData/AppCenter-ggabztdbmpzxaxfjxisclsawjhbn/Build/Intermediates.noindex/Sasquatch.build/Debug-iphonesimulator/SasquatchSwift Extension.build/Objects-normal/x86_64/Sasquatch-Swift.h
○ Target 'SasquatchSwift Extension' has compile command for Swift source files
→ Target 'AppCenterData iOS Framework' has copy command from '/Users/jaelim/Source/MobileCenter/AppCenter-SDK-Apple/AppCenterData/AppCenterData/MSData.h' to '/Users/jaelim/Library/Developer/Xcode/DerivedData/AppCenter-ggabztdbmpzxaxfjxisclsawjhbn/Build/Products/Debug-iphonesimulator/AppCenterData.framework/Headers/MSData.h'
○ Target 'AppCenterData iOS Framework' has target dependency on Target 'AppCenterAnalytics iOS Framework' due to target order in a “Target Dependencies” build phase or the scheme
```
*Cycle path: AppCenterAnalytics iOS Framework → AppCenterAuth iOS Framework → MSAL (iOS Static Library) → IdentityCore iOS → SasquatchSwift Extension → AppCenterData iOS Framework → AppCenterAnalytics iOS Framework*

**Pull request merged**